### PR TITLE
recreate project with svelte-kit 1.0 to fix build

### PR DESCRIPTION
**Pull request by Leaxe**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,20 @@
 {
 	"name": "time-budgeter",
-	"version": "0.0.1",
+	"version": "0.0.0",
```
Comment (silso) [inline package.json:3]: Match with version from changelog

**Review by silso — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Leaxe) [reply package.json:3]: i dont want to maintain the version in package.json and package-lock.json

Comment (silso) [reply package.json:3]: Why not

**Review by silso — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Leaxe) [reply package.json:3]: you need to run npm i every time you update the package.json version, and since its not a library it doesnt even matter

**Pull request merged**